codespell_lib/data/dictionary_code.txt

```diff
@@ -14,6 +14,7 @@ copyable->copiable
 creat->create, crate,
 define'd->defined
 deque->dequeue
+doas->does
```

Comment:
```suggestion
doas->does, do as,
```

Reply:
Pushed a new commit with that change, thanks. Sorry I missed it in your previous comment.

Reply:
No worries, I couldn't do a proper code suggestion on the deleted line.